skimage/restoration/_denoise.py

```diff
@@ -134,7 +134,7 @@ def denoise_tv_bregman(image, weight, max_iter=100, eps=1e-3, isotropic=True):
 
     Returns
     -------
-    u : ndarray
+    out : ndarray
         Denoised image.
```

Comment:
There is a reason this variable is called u. It refers to a variable in the equation further above.